src/CoWSwapEthFlow.sol

```diff
@@ -122,7 +122,19 @@ contract CoWSwapEthFlow is
     }
 
     /// @inheritdoc ICoWSwapEthFlow
-    function deleteOrder(EthFlowOrder.Data calldata order) external {
+    function deleteManyOrders(EthFlowOrder.Data[] calldata orderArray)
+        external
+    {
+        for (uint256 i = 0; i < orderArray.length; i++) {
+            deleteOrder(orderArray[i]);
+        }
+    }
+
+    /// @dev Marks an existing ETH flow order as invalid and refunds the trader of all ETH that hasn't been traded yet.
+    /// Note that some parameters of the order are ignored, as for example the order expiration date and the quote id.
+    ///
+    /// @param order The order to be deleted.
+    function deleteOrder(EthFlowOrder.Data calldata order) public {
```

Comment:
It would be nice to have this function in the interface as well, including comment (which is then inherited with `@inheritdoc ICoWSwapEthFlow`). This is so that `ICoWSwapEthFlow` can be always used as a drop-in replacement for these contract by projects that integrate with us.

Reply:
I imagine it's because the distinction between public and external isn't important for an interface: the interface specifies how other contracts can interact with this, and the public/external distinction only matters for the internal implementation of this contract.
Also, the Solidity docs [confirm](https://docs.soliditylang.org/en/latest/contracts.html#interfaces) that in interfaces `public` should be replaced by `external`. :white_check_mark: